scripts/paraformer/qnn/convert_predictor.sh

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+if [ -z $t ]; then
+  echo "Please run export t=num_input_seconds"
+  exit -1
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)